mobile-template-app/src/main/kotlin/cz/helu/templateapp/ui/dashboard/DashboardFragment.kt

```diff
@@ -8,4 +10,13 @@ import cz.helu.templateapp.databinding.FragmentDashboardBinding
 class DashboardFragment :
     BaseViewModelFragment<DashboardViewModel, FragmentDashboardBinding>(R.layout.fragment_dashboard) {
     override val viewModel: DashboardViewModel by viewModels()
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        setupToolbar()
+    }
+
+    private fun setupToolbar() {
+        toolbar?.title = getString(R.string.title_dashboard)
```

Comment:
I'd create a method in BaseFragment and probably make it required. If you call it you should now the toolbar is there.

```
fun setupToolbar(@StringRes titleId : Int) {
         toolbar!!.title = getString(R.string.title_dashboard) 
}
```